### PR TITLE
Add build number awareness

### PR DESCRIPTION
**Pull request by napstr**
Created the branch/opened the PR to get a build from the CI.
Confirming that it's working.
image

The build number can be seen with the ;;mstats command (also added the lavaplayer version to it because why not).

### CODE REVIEW

File: FredBoat/src/main/java/fredboat/FredBoat.java [hunk 1]
```diff
@@ -111,6 +112,11 @@ public static void main(String[] args) throws LoginException, IllegalArgumentExc
                 " |  __| '__/ _ \\/ _` |  _ < / _ \\ / _` | __|\n" +
                 " | |  | | |  __/ (_| | |_) | (_) | (_| | |_ \n" +
                 " |_|  |_|  \\___|\\__,_|____/ \\___/ \\__,_|\\__|\n\n");
+        log.info("App Info:"
+                + "\n\tGroup:    " + AppInfo.getAppInfo().GROUP_ID
+                + "\n\tArtifact: " + AppInfo.getAppInfo().ARTIFACT_ID
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/FredBoat.java]: Group and artefact seems redundant

File: FredBoat/src/main/java/fredboat/command/maintenance/StatsCommand.java [hunk 2]
```diff
@@ -74,7 +76,9 @@ public void onInvoke(CommandContext context) {
         str = str + "Known users in servers:         " + FredBoat.countAllUniqueUsers() + "\n";
         str = str + "Distribution:                   " + Config.CONFIG.getDistribution() + "\n";
         str = str + "JDA responses total:            " + context.guild.getJDA().getResponseTotal() + "\n";
-        str = str + "JDA version:                    " + JDAInfo.VERSION;
+        str = str + "JDA version:                    " + JDAInfo.VERSION + "\n";
+        str = str + "FredBoat version:               " + AppInfo.getAppInfo().getVersionBuild() + "\n";
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/command/maintenance/StatsCommand.java]: Neat, I was hoping you would do this

**Comment by Frederikam** [comment]
Before I merge, what happens if building outside of CI?

**Comment by napstr** [comment]
image
image

**Pull request merged**